solutions/77.js

```diff
@@ -18,4 +17,18 @@ const solution = (string) =>{
   return data;
 };
 
-module.exports = {solution};
+// Maricris Bonzo: @Seemcat
+const solution1 = (string) =>{
+  let i = 0;
+  let obj = {};
+  while (i < string.length){
+    if (string[i] === 'a' || 'e' || 'i' || 'o' || 'u'){
```

Comment:
Did you test your solution?  I don't think this if condition would work: `if (string[i] === 'a' || 'e' || 'i' || 'o' || 'u')`.